00-Login/Auth0Sample/HomeViewController.swift

```diff
@@ -25,28 +25,47 @@ import UIKit
 import Auth0
 
 class HomeViewController: UIViewController {
+    private var isLogged = false
```

Comment:
@benlizar Can we rename this to something a bit more clear? I'd suggest `isAuthenticated` or something similar.